ssl/quic/quic_tls.c

```diff
@@ -628,6 +641,20 @@ void ossl_quic_tls_free(QUIC_TLS *qtls)
     OPENSSL_free(qtls);
 }
 
+static int raise_error(QUIC_TLS *qtls, uint64_t error_code,
+                       const char *error_msg)
+{
+    qtls->error_code = error_code;
+    qtls->error_msg  = error_msg;
+    qtls->inerror    = 1;
+    return 0;
```

Comment:
Eventually we call ERR_raise_data when `ossl_quic_channel_raise_protocol_error` gets called....but by then we've lost the original file/line of the error. Shouldn't we be propagating the file/line of the error? E.g. see the definition of the SSLfatal macro which ensures `OPENSSL_FILE` and `OPENSSL_LINE` are captured. This probably applies everywhere else that `ossl_quic_channel_raise_protocol_error` gets called too